src/pages/UserPage.vue

```diff
@@ -68,7 +84,18 @@ async function loadUser() {
     });
 }
 
-onMounted(() => {
+/**
+ * Get user groups using user Id.
+ * @returns {Promise<void>} Promise with nothing on success.
+ */
+async function search() {
```

Comment:
rename to loadGroups
And add a new function search:
```js
function search() {
  loading.value = true;
  Promise.allSettled(
    loadUser(),
    loadGroups(),
  ).finaly(() => {
     loading.value = false;
  })
}
```

Reply:
done